packages/ui/src/theme/css/component/inAppMessaging/banner.scss

```diff
@@ -0,0 +1,62 @@
+.amplify-in-app-messaging-banner__button {
+  background-color: var(--amplify-components-inappmessagingbanner-button-background-color);
+  border-style: var(--amplify-components-inappmessagingbanner-button-border-style);
+  width: var(--amplify-components-inappmessagingbanner-button-width);
+}
+.amplify-in-app-messaging-banner__buttons-container {
+  margin-top: var(--amplify-components-inappmessagingbanner-buttons-container-margin-top);
+}
+.amplify-in-app-messaging-banner__container {
+  background-color: var(--amplify-components-inappmessagingbanner-banner-container-background-color);
+  box-shadow: var(--amplify-components-inappmessagingbanner-banner-container-box-shadow);
+  flex-direction: var(--amplify-components-inappmessagingbanner-banner-container-flex-direction);
+  margin: var(--amplify-components-inappmessagingbanner-banner-container-margin);
+  padding: var(--amplify-components-inappmessagingbanner-banner-container-padding);
+  position: var(--amplify-components-inappmessagingbanner-banner-container-position);
+  right: var(--amplify-components-inappmessagingbanner-banner-container-right);
+  row-gap: var(--amplify-components-inappmessagingbanner-banner-container-row-gap);
+  @media only screen and (min-width: 600px) {
+      max-width: 50%;
+      min-width: 400px;
+  }
+  @media only screen and (max-width: 600px) {
+      width: calc(100% - 32px);
+  }
+}
+.amplify-in-app-messaging-banner__content-container {
+  flex-direction: var(--amplify-components-inappmessagingbanner-content-container-flex-direction);
+  margin-bottom: var(--amplify-components-inappmessagingbanner-content-container-margin-bottom);
+}
+.amplify-in-app-messaging-banner__image {
+  object-fit: var(--amplify-components-inappmessagingbanner-image-object-fit);
+}
+.amplify-in-app-messaging-banner__image-container {
+  justify-content: var(--amplify-components-inappmessagingbanner-image-container-justify-content);
+  min-width: var(--amplify-components-inappmessagingbanner-image-container-min-width);
+  max-width: var(--amplify-components-inappmessagingbanner-image-container-max-width);
+  padding: var(--amplify-components-inappmessagingbanner-image-container-padding);
+  width: var(--amplify-components-inappmessagingbanner-image-container-width);
+}
+.amplify-in-app-messaging-banner__text-container {
+  flex: var(--amplify-components-inappmessagingbanner-text-container-flex);
+  flex-direction: var(--amplify-components-inappmessagingbanner-text-container-flex-direction);
+  gap: var(--amplify-components-inappmessagingbanner-text-container-gap);
+  justify-content: var(--amplify-components-inappmessagingbanner-text-container-justify-content);
+}
+.amplify-in-app-messaging-banner__top {
```

Comment:
Kinda nit but since `top` `middle` `bottom` are more modifiers to the banner, I think it's more conventional to use `--` instead of `__`